### DISCUSSION TITLE
Help Needed with reverification

### DISCUSSION THREAD

**Discussion by shashankx86** [community/community#61315] [GitHub Education]
### Hi everyone,

It asked to use school email ID but my school does not provide email, 
my reverification got rejected (1st attempt, I it was due not using full name on github acc), used ID card provided by school
Now i am doing reverification process again it won't accept that ID card photo, It said 

```
Your application cannot be reviewed until you fix the following:

    - If your school does not provide academic email then you must enable use of your device camera to capture your image.
    - Your image should be one of the following:
      - Dated school ID
 ```

I took photo of **Dated school ID** (Took photo then selected Proof type Dated school ID), Then It said 

```
Your application cannot be reviewed until you fix the following:

    - Your image should be one of the following:
      - Dated school ID

```

(jUST BE TO CLEAR i DIDN'T UPLOAD PHOTO I TOOK PHOTO NO UPLOAD)
What The hell I do, It asked **Dated school ID** photo, I gave it and it still saying provide **Dated school ID**

**Comment by 50bvd** [comment]
Hi!

As far as I remember, your school's email is required to take advantage of Git Studients, as it uses a domain that must be approved by GIT in order for students to benefit.

Concerning the photo that justifies your schooling in such school, you can send your student card, in my case I had sent directly a photo of my tuition contract. Whether it's taken with your phone or scanned in, the most important thing is that the content is legible.

Regarding your school's email address, I recommend that you check with your school's system administrator to see if he can create an exemption for you to have an email address.

**Reply by shashankx86** [reply]
It never asked me for email ID, and the only thing it asked is School ID

> I recommend that you check with your school's system administrator to see if he can create an exemption for you to have an email address.

Tried but, no luck

> Concerning the photo that justifies your schooling in such school, you can send your student card, in my case I had sent directly a photo of my tuition contract. Whether it's taken with your phone or scanned in

Thats what I'm doing (tho its not **tuition contract** its school ID) 

> the most important thing is that the content is legible.

It is

**Reply by 50bvd** [reply]
Has your school's adminsys gone through the procedure to have your school recognized by the GIT Studient program? 

This is necessary because if GIT doesn't have your school in its database, you won't be able to benefit from student advantages.

**Reply by 50bvd** [reply]
GIT generally accepts it with eyes closed if it's an it-related training course. But it's up to the system administrator to make the request to GIT, and when this is done, you as a student will be able to access the benefits.

**Reply by shashankx86** [reply]
School Name and URL was on reverification page

**Reply by 50bvd** [reply]
I see the pro label on your profile, did you pay for the GIT features, or do you have the student benefits since then?

If not, maybe 24/48 hours, if you still have nothing, send a message to GIT support directly. If your school is listed, you'll need to access the benefits.

**Reply by shashankx86** [reply]
> did you pay for the GIT features,

No

> do you have the student benefits since then?

Yes, Only 13 days left

**Reply by 50bvd** [reply]
I just found this thread -> https://github.com/orgs/community/discussions/59360

If you don't have an e-mail with your school's FQDN, you won't be able to take advantage of student benefits...

sorry for you

**Reply by shashankx86** [reply]
but 
![248873808-a000cf40-d7f4-406b-bb25-4f2f4279ec62](https://github.com/community/community/assets/64682801/9ee877d5-96b5-4370-8690-e4f74b042d00)

**Reply by 50bvd** [reply]
I won't be able to help you anymore, good luck dude!
